file-formats/enhs/enhs.json

```diff
@@ -36,15 +38,15 @@
           ],
           "enumDescriptions": [
             "Standard",
```

Comment:
Is "Standard" correct or is it "Default"? Not sure. Have seen default as well in the past.

Reply:
Do you mean that as ENHS specific or general? We used "Standard" also for all the other types.

Reply:
then it´s fine for me